config/google-maps/repos/joreilly-GalwayBus.json

```diff
@@ -0,0 +1,18 @@
+{
+  "expertise": "INTERMEDIATE",
+  "version": 1,
+  "source": "github",
+  "authorIds": [
+    "joreilly"
+  ],
+  "owner": "joreilly",
+  "repo": "GalwayBus",
+  "name": "GalwayBus",
+  "shortDescription": "Kotlin Multiplatform bus information related project using Jetpack Compose and SwiftUI.  Includes use of Maps Compose for showing maps with bus stops and realtime bus positions.",
+  "longDescription": "Kotlin Multiplatform bus information related project using Jetpack Compose and SwiftUI.  Includes use of Maps Compose for showing maps with bus stops and realtime bus positions.",
+  "content": "README.md",
+  "pages": [],
+  "tags": [
+    "maps-sdk-android"
```

Comment:
Additional Tags: Kotlin, Jetpack Compose

Reply:
Hi Swathi,

Jetpack Compose is not a valid tag in our repo.
You can take a look at valid tags [here](https://github.com/google/devlibrary/blob/c016e34c75ffa1bd07067be3582f28ace301649d/shared/product.ts#L296C1-L331)

Reply:
We do have Compose, I've added that